### ISSUE TITLE
Question: Use custom contract with tznft

### ISSUE THREAD

**Issue by jdsika** [oxheadalpha/nft-tutorial#80]
The tutorial states the command [create-collection](https://github.com/oxheadalpha/nft-tutorial/tree/master/packages/tznft#originate-nft-collection-contract) and I cannot find the definition of this contract.

There is the [tutorial](https://github.com/oxheadalpha/nft-tutorial/blob/master/packages/fa2-contracts/README.md) on how to create a custom modular contract but how do I specify to use this custom version?

**Comment by emishur** [comment]
With `nft-tutorial` you can create only one `pre-canned` version of the contract. Although the contract is created using [fa2-contracts](https://github.com/oxheadalpha/nft-tutorial/tree/master/packages/fa2-contracts) package (see the contract source [here](https://github.com/oxheadalpha/nft-tutorial/tree/master/packages/tznft/ligo)), `nft-tutorial` does not let you originate any arbitrary FA2 NFT contract at the moment.

**Comment by emishur** [comment]
@jdsika Yes, I am planning to add FA2.1 support, but waiting when the standard is stabilized. There were lots of changes since the original proposal.

**Comment by jdsika** [comment]
ok! What does your crystal ball say about the timeline here? :) 

I am looking at the discussions and cannot understand what the process is to find convergence here?

**Comment by emishur** [comment]
I think we are almost there. There is [FA2.1 draft](https://gitlab.com/tezos/tzip/-/blob/master/drafts/current/draft-fa2-update.md) uploaded to TZIP repo recently.
